blocks/footer/footer.css

```diff
@@ -16,5 +18,134 @@ footer .footer p {
 
 footer .footer-slogan {
   font-size: var(--footer-font-size-slogan);
+  padding-bottom: 40px;
+  font-style: italic;
+}
+
+footer li {
+  display: inline-block;
+  margin-right: 1rem;
+}
+
+footer a {
+  color: #fff;
+}
+
+footer li a:any-link {
+  color: white;
+}
+
+.footer-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.footer-container .right-side .icon{
+  fill: #949494;
+  transition: all 0.3s ease;
+  line-height: 1.6;
+  width: 32px;
+  height: 32px;
+  margin-bottom: 19px;
+}
+
+.footer-container [class^="footer-menu-list-"]{
+    padding-left:0;
+}
+
+.footer-container ul.footer-menu-list-0 li a {
+  font-size: 14px;
+  letter-spacing: 3px;
+}
+
+.footer-container ul.footer-menu-list-1 li a {
+  font-size: 14px;
+  opacity: 0.5;
+}
+
+
+.footer-container .right-side > p:nth-child(7) > a {
+  opacity: 0.5; 
+  color: #fff;
+ }
+
+.footer-container ul.footer-menu-list-1 li a:hover {
+  opacity: 1 !important;
```

Comment:
We should not use !important. see https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity